pyproject.toml

```diff
@@ -118,8 +118,9 @@ branch = true
 [tool.coverage.report]
 include = [
     "pytest_django/*",
-    "pytest_django_test/*",
-    "tests/*",
```

Comment:
It's actually highly discouraged to skip measuring tests:
https://web.archive.org/web/20250505160753/https://nedbatchelder.com/blog/201908/dont_omit_tests_from_coverage.html

Reply:
There should be a Codecov check that demands 100% coverage on the tests so that it's possible to reliably tell that there's no accidentally skipped tests or parts of the testing infra. Otherwise, there's a perception that things are being tested when they aren't.

Reply:
We're headed that direction, its anyone's guess when we arrive 😇